### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by jjw24**

### CODE REVIEW

**Review by taooceros — APPROVED** [review]

**Comment by taooceros** [comment]
I check about whether some string is deleted, and that's true. However, it seems like it is because we modify the original string?

File: Flow.Launcher/Languages/es-419.xaml [hunk 1]
```diff
@@ -69,7 +70,7 @@
     <system:String x:Key="currentPriority">Prioridad Actual</system:String>
     <system:String x:Key="newPriority">Nueva Prioridad</system:String>
     <system:String x:Key="priority">Prioridad</system:String>
-    <system:String x:Key="priorityToolTip">Cambiar la prioridad del resultado del plugin</system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/es-419.xaml:72]: This shouldn't be deleted?

Comment (taooceros) [reply Flow.Launcher/Languages/es-419.xaml:72]: The original string is updated, so i am not sure whether this is correct?

Comment (jjw24) [reply Flow.Launcher/Languages/es-419.xaml:72]: Unless the translation has been updated in Crowdin, I think we should align these translation with this PR https://github.com/Flow-Launcher/Flow.Launcher/pull/1410/files

Comment (taooceros) [reply Flow.Launcher/Languages/es-419.xaml:72]: ok

**Review by kubalav — APPROVED** [review]

**Comment by taooceros** [comment]
The latest chinese translation doesn't seem Correct. I will adjust that a bit before merging.

**Review by taooceros — APPROVED** [review]

**Pull request merged**